app/directive/social-icons.html

```diff
@@ -0,0 +1,10 @@
+<nav>
+  <img ng-src="../../assets/SocialIcons.png">
+  <h2>{{ socialIconsCtrl.tester }}</h2>
+  <ul>
+    <li class="social-icon" ng-repeat="icon in socialIconsCtrl.icons">{{ icon }}</li>
+    <li class="socialIcons-twitter"></li>
+    <li class="socialIcons-facebook"></li>
+    <li class="socialIcons-instagram"></li>
```

Comment:
Good job getting the icons to display. You should be using the ng-repeat to display the icons though.